SDK v2 migration guide.md

```diff
@@ -52,6 +50,7 @@ to migrate to version 2.x when they have the chance. For more details on LTS rel
   - It had very limited support across the device options and some APIs behaved differently.
 - Some options that were previously set in the `DeviceClient` constructor are now in the optional `IotHubClientOptions` parameter.
 - This method has been split into the three individual steps that this method used to take. See [this file upload sample](./iothub/device/samples/getting%20started/FileUploadSample/) for an example of how to do file upload using these discrete steps.
+- Cloud to device messages can only be received by calling ```SetReceiveMessageHandlerAsync``` and providing a callback. Users can no longer poll for messages with ```ReceiveAsync```.
```

Comment:
Triple back ticks are only needed for code blocks. Code snippets can be single back ticks.

```suggestion
- Cloud to device messages can only be received by calling `SetReceiveMessageHandlerAsync` and providing a callback. Users can no longer poll for messages with `ReceiveAsync`.
```